.github/workflows/actions/merge-cov/action.yml

```diff
@@ -33,9 +33,18 @@ runs:
   # different parallelized runs
   - name: Merge coverage reports
     shell: bash
+    # NOTE: Merging code reports generated from self-hosted (aml) runners can be problematic
+    #       as reports may reference the source code at an unresolvable absolute path.
+    #       For example, you may encounter errors like:
+    #            "CoverageWarning: Couldn't parse 
+    #            '/home/azureuser/runner/work/recommenders/recommenders/recommenders/evaluation/__init__.py':
+    #            No source for code"
+    # Work-around: Creating a symlink at the root that points to the default local runner folder: '/home/runner/work/recommenders'
     run: |
+      sudo mkdir -p /home/azureuser
+      sudo ln -s /home/runner /home/azureuser/runner
```

Comment:
/home/runner may not exist, don't you need to create it first?

Reply:
`/home/runner` is a default path that should be present as long as we are using github-hosted runners to run this step. If that path doesn't exist, it will be a different but big problem (a lot of things can break for github action alone). Of course, there's no guarantee that this wouldn't change, but I think this is relatively safe assumption to make it simpler, unless you think otherwise.

Reply:
Ok, let's see if it works.

Reply:
hmm but we get 23% coverage, did we change the sets of test we run?